hyperspy/conftest.py

```diff
@@ -1,12 +1,14 @@
 # Configure mpl and traits to work in a headless system
 from traits.etsconfig.api import ETSConfig
 ETSConfig.toolkit = "null"
-import matplotlib
-matplotlib.use("Agg")
+
+# pytest-mpl 0.7 already import pyplot, so setting the matplotlib backend to 
+# 'agg' as early as we can is useless for testing.
+import matplotlib.pyplot as plt
 
 import pytest
 import numpy as np
-import matplotlib.pyplot as plt
+import matplotlib
```

Comment:
It is used in the `mpl_cleanup` pytest fixture, am i missing something obvious?!